README.rst

```diff
@@ -122,7 +122,7 @@ for academic publications:
 Join us
 -------
 
-`ixdat`` is free and open source software and we welcome input and new collaborators. Please help us improve!
+``ixdat`` is free and open source software and we welcome input and new collaborators. Please help us improve!
```

Comment:
"Please help us improve ixdat!" maybe

Reply:
yes, haha. 
May not be the right place to ask for more general help ;)

Reply:
Nah, well, it was more like I think it sounded a bit like we have no idea what we are doing, which I think is not quite right ;)